src/main/java/org/opentripplanner/middleware/otp/response/Itinerary.java

```diff
@@ -77,14 +69,9 @@ public class Itinerary implements Cloneable {
     public Double elevationGained = 0.0;
 
     /**
-     * The number of transfers this trip has.
-     */
-    public Integer transfers = 0;
-
-    /**
-     * Fare information for this itinerary.
+     * How many transfers are part of this itinerary.
      */
-    public FareWrapper fare;
+    public Integer numberOfTransfers = 0;
```

Comment:
Keep the previous name `transfers`. In the default graphQL query we are using (set in class `GraphQLUtils`), we write:
```
transfers: numberOfTransfers
```
which renames `numberOfTransfers` back to `transfers` . `transfers` is used in the Web/mobile UIs to sort itineraries.

Reply:
@binh-dam-ibigroup this was renamed to match the spec! I've added `@JsonProperty("numberOfTransfers")` so the original value can be kept.